src/Compilers/CSharp/Test/Emit2/Semantics/PatternSwitchTests.cs

```diff
@@ -1231,7 +1232,7 @@ public static void M(object o)
 0.0m !
 decimal {2.1m}
 1.0m !
-null";
+null");
```

Comment:
In the code which creates the unit tests the current culture is the machine culture. Have to use `Invariant` formatting here to ensure the baseline is generated in an invariant fashion as that is how the test is executed.